lib/inspec/resources/sys_info.rb

```diff
@@ -13,13 +13,38 @@ class System < Inspec.resource(1)
       describe sys_info do
         its('hostname') { should eq 'example.com' }
       end
+
+      describe sys_info do
+        its('hostname') { should eq 'example.com' }
```

Comment:
Please add entries to `docs/resources/sys_info.md.erb` explaining the new options and listing the choices that are recognized.